client/my-sites/preview/controller.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import WebPreviewContent from 'components/web-preview/content';
+
+export default {
+	preview: function( context, next ) {
+		context.primary = (
+			<div style={ { height: '100%' } }>
+				<WebPreviewContent
+					previewUrl={ `https://${ context.params.site }/?iframe=true&preview=true` }
```

Comment:
Is `context.params` guaranteed to exist & is `context.params.site` guaranteed to be a URL, somewhere?

If not, we should probably add some guarding